spec/support/kafka.rb

```diff
@@ -0,0 +1,63 @@
+module KafkaHelper
```

Comment:
just copied out of promiscuous so we aren't coupled to promiscuous spec helpers except for commented line